packages/language-server/test/plugins/typescript/features/FindFileReferencesProvider.test.ts

```diff
@@ -0,0 +1,68 @@
+import * as assert from 'assert';
+import * as path from 'path';
+import ts from 'typescript';
+import { Location, Position, Range } from 'vscode-languageserver';
+import { Document, DocumentManager } from '../../../../src/lib/documents';
+import { LSConfigManager } from '../../../../src/ls-config';
+import { FindFileReferencesProviderImpl } from '../../../../src/plugins/typescript/features/FindFileReferencesProvider';
+import { LSAndTSDocResolver } from '../../../../src/plugins/typescript/LSAndTSDocResolver';
+import { pathToUrl } from '../../../../src/utils';
+
+const testDir = path.join(__dirname, '..');
+
+function test(useNewTransformation: boolean) {
+    return () => {
+        function getFullPath(filename: string) {
+            return path.join(testDir, 'testfiles', filename);
+        }
+        function getUri(filename: string) {
+            const filePath = path.join(testDir, 'testfiles', filename);
+            return pathToUrl(filePath);
+        }
+
+        function setup(filename: string) {
+            const docManager = new DocumentManager(
+                (textDocument) => new Document(textDocument.uri, textDocument.text)
+            );
+            const lsConfigManager = new LSConfigManager();
+            lsConfigManager.update({ svelte: { useNewTransformation } });
+            const lsAndTsDocResolver = new LSAndTSDocResolver(
+                docManager,
+                [testDir],
+                lsConfigManager
+            );
+            const provider = new FindFileReferencesProviderImpl(lsAndTsDocResolver);
+            const document = openDoc(filename);
+            return { provider, document };
+
+            function openDoc(filename: string) {
+                const filePath = getFullPath(filename);
+                const doc = docManager.openDocument(<any>{
+                    uri: pathToUrl(filePath),
+                    text: ts.sys.readFile(filePath) || ''
+                });
+                return doc;
+            }
+        }
+
+        async function test() {
+            const { provider, document } = setup('find-file-references-child.svelte');
+
+            const results = await provider.fileReferences(document.uri.toString());
```

Comment:
This was the call which yields no results.

Reply:
I got a hint of a comment elsewhere and tried loading the parent file as well and it worked.